ui/jest.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  testPathIgnorePatterns: [
+    "<rootDir>/build/",
+    "<rootDir>/node_modules/"
```

Comment:
Nit: mixing of `'` and `"`

Reply:
Yeeeah this project has a mix of quote styles throughout--one reason I'd really like to add Prettier